app.json

```diff
@@ -0,0 +1,8 @@
+{
+    "cron": [
+        {
+            "command": "python manage.py runjobs daily |& systemd-cat -t actions-registry",
+            "schedule": "@daily"
```

Comment:
I'd be tempted to add all the different schedules here, then a future developer doesn't have to know whether they're set up, they can focus on adding the Job instead.